src/Tribe/Main.php

```diff
@@ -17,7 +17,7 @@ class Tribe__Main {
 	const OPTIONNAME          = 'tribe_events_calendar_options';
 	const OPTIONNAMENETWORK   = 'tribe_events_calendar_network_options';
 
-	const VERSION             = '4.9.17';
+	const VERSION             = '4.9.18';
```

Comment:
:warning: Equals sign not aligned correctly; expected 1 space but found 13 spaces

<sub>hash: 1eb125f5345e78d2763bd5b0802a3232</sub>